test/e2e/network/networking_perf.go

```diff
@@ -37,6 +42,157 @@ const (
 	smallClusterTimeout = 200 * time.Second
 )
 
+func iperf3ServerDeployment(client clientset.Interface, namespace string) (*appsv1.Deployment, error) {
```

Comment:
thanks for doing this, this test looks much better then the ones we had
previously from what you posted in the other issue.

Reply:
👍